client/packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -109,15 +109,15 @@ const outboundParsers = {
       numberOfPacks: line.numberOfPacks,
       stockLineId: line.stockLine?.id ?? '',
       invoiceId: line.invoiceId,
-      totalBeforeTax: get.stockLineSubtotal(line),
+      totalBeforeTax: get.invoiceLineSubtotal(line),
```

Comment:
Would assume we are removing this in downstream branch

Reply:
https://github.com/msupply-foundation/open-msupply/issues/4763